package.json

```diff
@@ -33,19 +33,19 @@
     "gulp-connect": "^2.0.6",
     "gulp-filter": "^2.0.0",
     "gulp-highlight": "0.0.3",
-    "gulp-karma": "0.0.4",
     "gulp-load-plugins": "^0.8.0",
     "gulp-markdown": "^1.0.0",
     "gulp-minify-css": "^1.2.2",
     "gulp-ng-html2js": "^0.1.8",
     "gulp-prompt": "^0.1.2",
     "gulp-rename": "^1.2.0",
     "gulp-rsync": "^0.0.4",
-    "gulp-ruby-sass": "1.0.0-alpha",
-    "gulp-sass": "^1.3.3",
+    "gulp-ruby-sass": "^2.0.6",
```

Comment:
You don't need to make that change with this PR if you don't want to.  I've created an issue to address it https://github.com/base-apps/angular-base/issues/6.